docs/controller-skills/military.md

```diff
@@ -69,10 +69,51 @@ These Restricted Areas may be activated in vatSys, and the controllers of these
 Unless otherwise stated in Local Instructions, Restricted Areas activated for control by Military TCUs and Towers, will become **Class C** airspace.
 
 ## Formation Flights
+### Standard Formation
+A **Standard formation** is defined by more than one aircraft, separated by no more than **1nm** laterally/longitudinally, and no more than **100 feet** vertically, from the lead aircraft. All aircraft in the formation are treated as *one*. Ensure only the leader is squawking Mode C.
+
+!!! note
+    RAAF FIHA AD2 Supps and TERMAs can be found [here](https://ais-af.airforce.gov.au/australian-aip){target=new}
+
+!!! phraseology
+    **VIPR**: "Viper 1, 2, 3, 4, Brisbane Centre, Viper, Maintaining F280"  
+    **ARA**:  "Viper, Brisbane Centre" 
```

Comment:
I know that the first part of this transmission is each aircraft checking in on the new frequency, but I'm wondering if it might confuse controllers to see it displayed like this. Should we start with the transmission with just 'Brisbane Centre, Viper...' and leave the check in out? I can see controllers interpreting it as the lead aircraft identifying all of the aircraft which form part of the formation, rather than separate transmissions from four different aircraft.

Reply:
Thats what I had initially before alphadelta modified it + only check in if its not a plannned com switch so I think the orginal idea was better.

Reply:
Ah my bad, some crossed wires then. We'll discuss on our end

Reply:
> Thats what I had initially before alphadelta modified it + only check in if its not a plannned com switch so I think the orginal idea was better.

Formations do require a full check in for all members for initial contact with all civil units, whether the comm switch is planned or not. That being said, I have added a clarification line above it to establish what is going on and avoid any confusion for ATC